### ISSUE TITLE
Garbled text + crash with Iris 1.6.3

### ISSUE THREAD

**Issue by terzag** [IrisShaders/Iris#1994]
### Minecraft Version

Minecraft 1.19.4

### Iris Version

iris-mc1.19.4-1.6.3.jar

### Sodium Version

sodium-fabric-mc1.19.4-0.4.11+build.26.jar

### Operating System

Linux Xubuntu 23.04

### What is your GPU?

AMD Radeon RX 6500 XT

### Java Version

OpenJDK 17.0.6

### Reproduction Steps

When using Iris 1.16.3, I get garbled text (squares instead of characters) in the menu + some alert that I can't read as seen here: https://imgur.com/a/YWkyzZj
My game is in french, if that matters.

If I then try to load a world, the game crashes (see report).

Everything works fine with Iris 1.6.2 which, for some reason, has been removed from Modrinth (it's still available for Minecraft 1.18.2 and 1.19.2, not 1.19.4).

### Crash Report file and latest.log

```shell
---- Minecraft Crash Report ----
// Hey, that tickles! Hehehe!

Time: 2023-05-11 20:27:25
Description: Unexpected error

java.lang.IllegalStateException: An invalid version of Sodium is installed, and the warning screen somehow didn't work. This is a bug! Please report it to the Iris developers.
	at net.minecraft.class_761.handler$bbp000$iris$beginLevelRender(class_761.java:7447)
	at net.minecraft.class_761.method_22710(class_761.java:1232)
	at net.minecraft.class_757.method_3188(class_757.java:1100)
	at net.minecraft.class_757.method_3192(class_757.java:872)
	at net.minecraft.class_310.method_1523(class_310.java:1218)
	at net.minecraft.class_310.method_1514(class_310.java:801)
	at net.minecraft.client.main.Main.main(Main.java:237)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:462)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Render thread
Stacktrace:
	at net.minecraft.class_761.handler$bbp000$iris$beginLevelRender(class_761.java:7447)
	at net.minecraft.class_761.method_22710(class_761.java:1232)
	at net.minecraft.class_757.method_3188(class_757.java:1100)

-- Affected level --
Details:
	All players: 1 total; [class_746['abelthorne'/431, l='ClientLevel', x=-79.89, y=202.00, z=25.93]]
	Chunk stats: 1024, 612
	Level dimension: minecraft:overworld
	Level spawn location: World: (0,125,0), Section: (at 0,13,0 in 0,7,0; chunk contains blocks 0,-64,0 to 15,319,15), Region: (0,0; contains chunks 0,0 to 31,31, blocks 0,-64,0 to 511,319,511)
	Level time: 43758341 game time, 66055790 day time
	Server brand: fabric
	Server type: Integrated singleplayer server
Stacktrace:
	at net.minecraft.class_638.method_8538(class_638.java:455)
	at net.minecraft.class_310.method_1587(class_310.java:2406)
	at net.minecraft.class_310.method_1514(class_310.java:825)
	at net.minecraft.client.main.Main.main(Main.java:237)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:462)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)

-- Last reload --
Details:
	Reload number: 1
	Reload reason: initial
	Finished: No
	Packs: vanilla, Fabric Mods, file/Botany's+3D+Crops.zip, file/Botany's+3D+Mushrooms.zip, file/Mizunos 16 Craft JE_1.19.4-1.1.zip, file/Invisible Item Frame Pack_1.19.4-1.1.zip, file/Mizunos 16 Craft JE CIT_1.19.4-1.1.zip, file/Bibliophilia+1.3.zip, file/xali's+Potions+v1.0.zip, file/Brewing+Guide.zip, file/Fancy,+GUI+Overhaul+v0.1.2.7.zip, file/Fancy+Boss+Bars+v0.1.1.zip, resourcePack.continuity.default.name, resourcePack.continuity.glass_pane_culling_fix.name
	Recovery: Yes
	Recovery reason: java.util.concurrent.CompletionException: java.lang.BootstrapMethodError: java.lang.RuntimeException: Mixin transformation of net.minecraft.class_7764 failed
	at java.base/java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:315)
	at java.base/java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:320)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1770)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.exec(CompletableFuture.java:1760)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165)
Caused by: java.lang.BootstrapMethodError: java.lang.RuntimeException: Mixin transformation of net.minecraft.class_7764 failed
	at net.minecraft.class_7948$class_7949.method_47674(class_7948.java:22)
	at net.minecraft.class_7954.method_47683(class_7954.java:32)
	at net.minecraft.class_7954.md83591e$iris$lambda$iris$modifyForEachAction$0$0(class_7954.java:529)
	at java.base/java.util.TreeMap.forEach(TreeMap.java:1282)
	at net.minecraft.class_7954.method_47673(class_7954.java:30)
	at net.minecraft.class_7947.method_47669(class_7947.java:61)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at net.minecraft.class_7947.method_47667(class_7947.java:61)
	at net.minecraft.class_7766.method_47660(class_7766.java:113)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768)
	... 6 more


-- System Details --
Details:
	Minecraft Version: 1.19.4
	Minecraft Version ID: 1.19.4
	Operating System: Linux (amd64) version 6.2.0-20-generic
	Java Version: 17.0.3, Microsoft
	Java VM Version: OpenJDK 64-Bit Server VM (mixed mode), Microsoft
	Memory: 635523504 bytes (606 MiB) / 1979711488 bytes (1888 MiB) up to 2147483648 bytes (2048 MiB)
	CPUs: 8
	Processor Vendor: AuthenticAMD
	Processor Name: AMD Ryzen 5 1500X Quad-Core Processor
	Identifier: AuthenticAMD Family 23 Model 1 Stepping 1
	Microarchitecture: Zen
	Frequency (GHz): -0.00
	Number of physical packages: 1
	Number of physical CPUs: 4
	Number of logical CPUs: 8
	Graphics card #0 name: Navi 24 [Radeon RX 6400 / 6500 XT]
	Graphics card #0 vendor: Advanced Micro Devices, Inc. [AMD/ATI] (0x1002)
	Graphics card #0 VRAM (MB): 258.00
	Graphics card #0 deviceId: 0x743f
	Graphics card #0 versionInfo: unknown
	Virtual memory max (MB): 28128.48
	Virtual memory used (MB): 9246.76
	Swap memory total (MB): 18160.24
	Swap memory used (MB): 250.50
	JVM Flags: 8 total; -Xss1M -Xmx2G -XX:+UnlockExperimentalVMOptions -XX:+UseG1GC -XX:G1NewSizePercent=20 -XX:G1ReservePercent=20 -XX:MaxGCPauseMillis=50 -XX:G1HeapRegionSize=32M
	Fabric Mods: 
		advancementinfo: AdvancementInfo 1.19.4-fabric0.76.0-1.3.1
			cloth-config: Cloth Config v10 10.0.96
				cloth-basic-math: cloth-basic-math 0.6.1
		cem: Custom Entity Models 0.7.3
			completeconfig-base: completeconfig-base 2.3.1
			completeconfig-gui-cloth: completeconfig-gui-cloth 2.3.1
		citresewn: CIT Resewn 1.1.3+1.19.4
			citresewn-defaults: CIT Resewn: Defaults 1.1.3+1.19.4
		continuity: Continuity 3.0.0-beta.2+1.19.3
		entity_texture_features: Entity Texture Features 4.4.4
			org_apache_httpcomponents_httpmime: httpmime 4.5.10
		entityculling: EntityCulling-Fabric 1.6.2-mc1.19.4
		fabric-api: Fabric API 0.80.0+1.19.4
			fabric-api-base: Fabric API Base 0.4.25+504944c8f4
			fabric-api-lookup-api-v1: Fabric API Lookup API (v1) 1.6.26+504944c8f4
			fabric-biome-api-v1: Fabric Biome API (v1) 13.0.8+348a9c64f4
			fabric-block-api-v1: Fabric Block API (v1) 1.0.7+e022e5d1f4
			fabric-blockrenderlayer-v1: Fabric BlockRenderLayer Registration (v1) 1.1.35+c2e6f674f4
			fabric-client-tags-api-v1: Fabric Client Tags 1.0.16+504944c8f4
			fabric-command-api-v1: Fabric Command API (v1) 1.2.28+f71b366ff4
			fabric-command-api-v2: Fabric Command API (v2) 2.2.7+504944c8f4
			fabric-commands-v0: Fabric Commands (v0) 0.2.45+df3654b3f4
			fabric-containers-v0: Fabric Containers (v0) 0.1.55+df3654b3f4
			fabric-content-registries-v0: Fabric Content Registries (v0) 3.5.10+ae0966baf4
			fabric-convention-tags-v1: Fabric Convention Tags 1.4.2+9a7c5daaf4
			fabric-crash-report-info-v1: Fabric Crash Report Info (v1) 0.2.16+aeb40ebef4
			fabric-data-generation-api-v1: Fabric Data Generation API (v1) 11.4.1+504944c8f4
			fabric-dimensions-v1: Fabric Dimensions API (v1) 2.1.46+7f87f8faf4
			fabric-entity-events-v1: Fabric Entity Events (v1) 1.5.14+504944c8f4
			fabric-events-interaction-v0: Fabric Events Interaction (v0) 0.5.0+76ba65ebf4
			fabric-events-lifecycle-v0: Fabric Events Lifecycle (v0) 0.2.53+df3654b3f4
			fabric-game-rule-api-v1: Fabric Game Rule API (v1) 1.0.34+a1ccd7bff4
			fabric-item-api-v1: Fabric Item API (v1) 2.1.18+504944c8f4
			fabric-item-group-api-v1: Fabric Item Group API (v1) 3.0.6+043f9acff4
			fabric-key-binding-api-v1: Fabric Key Binding API (v1) 1.0.34+504944c8f4
			fabric-keybindings-v0: Fabric Key Bindings (v0) 0.2.32+df3654b3f4
			fabric-lifecycle-events-v1: Fabric Lifecycle Events (v1) 2.2.16+504944c8f4
			fabric-loot-api-v2: Fabric Loot API (v2) 1.1.28+75e98211f4
			fabric-loot-tables-v1: Fabric Loot Tables (v1) 1.1.32+9e7660c6f4
			fabric-message-api-v1: Fabric Message API (v1) 5.1.2+504944c8f4
			fabric-mining-level-api-v1: Fabric Mining Level API (v1) 2.1.40+49abcf7ef4
			fabric-models-v0: Fabric Models (v0) 0.3.31+504944c8f4
			fabric-networking-api-v1: Fabric Networking API (v1) 1.3.2+504944c8f4
			fabric-networking-v0: Fabric Networking (v0) 0.3.42+df3654b3f4
			fabric-object-builder-api-v1: Fabric Object Builder API (v1) 7.0.4+504944c8f4
			fabric-particles-v1: Fabric Particles (v1) 1.0.24+f1e4495bf4
			fabric-recipe-api-v1: Fabric Recipe API (v1) 1.0.9+a1ccd7bff4
			fabric-registry-sync-v0: Fabric Registry Sync (v0) 2.2.1+504944c8f4
			fabric-renderer-api-v1: Fabric Renderer API (v1) 2.2.6+504944c8f4
			fabric-renderer-indigo: Fabric Renderer - Indigo 1.1.2+81e8c576f4
			fabric-renderer-registries-v1: Fabric Renderer Registries (v1) 3.2.39+df3654b3f4
			fabric-rendering-data-attachment-v1: Fabric Rendering Data Attachment (v1) 0.3.29+afca2f3ef4
			fabric-rendering-fluids-v1: Fabric Rendering Fluids (v1) 3.0.22+504944c8f4
			fabric-rendering-v0: Fabric Rendering (v0) 1.1.42+df3654b3f4
			fabric-rendering-v1: Fabric Rendering (v1) 2.1.2+504944c8f4
			fabric-resource-conditions-api-v1: Fabric Resource Conditions API (v1) 2.3.2+e6c7d4eef4
			fabric-resource-loader-v0: Fabric Resource Loader (v0) 0.11.3+1e1fb126f4
			fabric-screen-api-v1: Fabric Screen API (v1) 1.0.46+504944c8f4
			fabric-screen-handler-api-v1: Fabric Screen Handler API (v1) 1.3.21+504944c8f4
			fabric-sound-api-v1: Fabric Sound API (v1) 1.0.10+504944c8f4
			fabric-transfer-api-v1: Fabric Transfer API (v1) 3.1.2+504944c8f4
			fabric-transitive-access-wideners-v1: Fabric Transitive Access Wideners (v1) 3.0.4+504944c8f4
		fabricloader: Fabric Loader 0.14.19
		freecam: Freecam (Modrinth Edition) 1.1.9+1.19
		indium: Indium 1.0.17+mc1.19.4
		iris: Iris 1.6.3
			io_github_douira_glsl-transformer: glsl-transformer 2.0.0-pre13
			org_anarres_jcpp: jcpp 1.4.14
			org_antlr_antlr4-runtime: antlr4-runtime 4.11.1
		java: OpenJDK 64-Bit Server VM 17
		lithium: Lithium 0.11.1
		minecraft: Minecraft 1.19.4
		modmenu: Mod Menu 6.2.2
		no-name-tags: No Name Tags 1.1.0
		phosphor: Phosphor 0.8.1
		simple_durability_tooltip: Simple Durability Tooltip 2.0.0
		sodium: Sodium 0.4.11+build.26
		statuseffecttimer: Status Effect Timer 1.1.1+1.19
		xaerominimap: Xaero's Minimap 23.4.1
		xaeroworldmap: Xaero's World Map 1.30.1
	Loaded Shaderpack: ComplementaryReimagined_r2.1.zip
		Profile: HIGH (+0 options changed by user)
	NEC status: No NEC detected
	Launched Version: fabric-loader-0.14.19-1.19.4
	Backend library: LWJGL version 3.3.1 SNAPSHOT
	Backend API: AMD Radeon RX 6500 XT (navi24, LLVM 15.0.7, DRM 3.49, 6.2.0-20-generic) GL version 4.6 (Core Profile) Mesa 23.0.2 (git-4d5e73870e), AMD
	Window size: 1920x1080
	GL Caps: Using framebuffer using OpenGL 3.2
	GL debug messages: 
	Using VBOs: Yes
	Is Modded: Definitely; Client brand changed to 'fabric'; Server brand changed to 'fabric'
	Type: Integrated Server (map_client.txt)
	Graphics mode: fancy
	Resource Packs: 
	Current Language: fr_fr
	CPU: 8x AMD Ryzen 5 1500X Quad-Core Processor
	Server Running: true
	Player Count: 1 / 8; [class_3222['abelthorne'/431, l='ServerLevel[Nouveau monde]', x=-79.89, y=202.00, z=25.93]]
	Data Packs: vanilla, fabric
	Enabled Feature Flags: minecraft:vanilla
	World Generation: Stable
```

**Comment by purple512** [comment]
Iris 1.6.3 requires Sodium 0.4.10.

Sodium 0.4.11 was pulled because of stability issues, and that is why Iris 1.6.3 was released, so that you can use Sodium 0.4.10 (the now most recent version) with the most recent Iris.

**Comment by terzag** [comment]
Oh, ok. I saw that Iris 1.6.3 mentioned Sodium 0.4.10 compatibility but I though it was in addition to 0.4.11, I didn't realize that 0.4.11 had been removed.

I switched back to Sodium 0.4.10 (as well as corresponding Indium) and Iris 1.6.3 and it works properly. Thanks.
